playbooks/files/cpu_load.py

```diff
@@ -42,15 +40,16 @@ def main():
 
     utilization = []
     with open(args.source, 'r') as file:
-        for line in file:
-            if line.strip():
-                try:
-                    level = float(line)
-                    if level < 0 or level > 100:
-                        raise ValueError
-                    utilization.append(int(level))
-                except ValueError:
-                    LOGGER.error("the source file must only contain new line separated numbers in the [0, 100] range")
+        levels = file.read().splitlines()
+    file.close()
+    for line in levels:
+        if line.strip():
+            try:
+                if int(line) < 0 or int(line) > 100:
+                    raise ValueError
```

Comment:
why use exception here? You can just use `if... else...`

Reply:
just for notification